07/27/2024 04:41:02 PM Logging starts
Traceback (most recent call last):
  File "./popular_projects_snippets_dataset/black/bodies/body_175_refine_3_0.py", line 44, in <module>
    for _index, leaf, leaf_length in line.enumerate_with_length():
ValueError: not enough values to unpack (expected 3, got 2)
Command exited with non-zero status 1
1.29user 3.32system 0:00.86elapsed 532%CPU (0avgtext+0avgdata 105356maxresident)k
0inputs+40outputs (0major+24195minor)pagefaults 0swaps
